modules/dashboard/src/components/StatsSummary.js

```diff
@@ -116,78 +127,124 @@ const StatsSummary = ({ classes, messaging }) => {
       const res = await messaging.getStateChannelByUserPubId(xPub);
       let balanceArr = [];
       res.freeBalanceAppInstance.latestState.balances[0].forEach(balance => {
-        if (balance.to === address.mainnet) {
+        if (balance.to === address.staging) {
           balanceArr.push(parseInt(balance.amount._hex, 16));
         }
       });
 
       const balanceArrReduced = balanceArr.reduce((a, b) => {
         return a + b;
       }, 0);
-
       return balanceArrReduced;
     } catch (e) {
       setSearchError(`error getting channel: ${e}`);
     }
   };
 
   const getTransfers = async () => {
-    const res = await messaging.getAllLinkedTransfers() || [];
+    const res = (await messaging.getAllLinkedTransfers()) || [];
 
     let totalTransfers = [];
+    let pastDayTotal = 0,
+      pastWeekTotal = 0,
+      pastMonthTotal = 0;
     if (res) {
       for (let transfer of res) {
         totalTransfers.push(parseInt(transfer.amount._hex, 16));
+        const createdDate = new Date(transfer.createdAt);
+        const hourDifference = (Date.now() - createdDate.getTime()) / 3600000;
+        if (hourDifference <= 24) {
+          pastDayTotal++;
+        } else if (hourDifference > 24 && hourDifference <= 168) {
+          pastWeekTotal++;
+        } else if (hourDifference > 168 && hourDifference <= 720) {
+          pastMonthTotal++;
+        }
```

Comment:
why would the past day total not be reflected in the past week or past month total?

Reply:
*facepalm*

Reply:
🙃

Reply:
resolving, addressing in #690 so not blocking